src/surfaces/actionBar/actionBar.jsx

```diff
@@ -308,7 +308,7 @@ class ActionBar extends React.PureComponent {
                                                 <Button
                                                     color={button.color}
                                                     designVersion={1}
-                                                    disabled={!!button.disabled}
+                                                    disable={!!button.disable || !!button.disabled}
```

Comment:
I wonder if we should just keep this `disabled`. One reason, designVersion `1` supports it and two, designVersion `2` requires it. Up to you.

Reply:
I'm good either way.  I've always favored `disabled` for consistency with native DOM and most standards I am aware of.
I just want whatever:
a) won't break stuff, and
b) won't log too many warnings! =)

Reply:
B convinces me. I hate logs/warnings.